test/common/schemaform/fields/ArrayField.unit.spec.jsx

```diff
@@ -53,8 +55,44 @@ describe('Schemaform <ArrayField>', () => {
 
     expect(tree.subTree('TitleField').props.title).to.equal(uiSchema['ui:title']);
     expect(tree.everySubTree('SchemaField')).not.to.be.empty;
+    expect(tree.everySubTree('.va-growable-background')).to.be.empty;
   });
   it('should render items', () => {
+    const idSchema = {
+      $id: 'field'
+    };
+    const schema = {
+      type: 'array',
+      items: [],
+      additionalItems: {
+        type: 'object',
+        properties: {
+          field: {
+            type: 'string'
+          }
+        }
+      }
+    };
+    const uiSchema = {
+      'ui:title': 'List of things',
+      'ui:options': {
+        viewField: f => f
+      }
+    };
+    const tree = SkinDeep.shallowRender(
+      <ArrayField
```

Comment:
Interesting, form data is not being provided here, since it is testing an empty form. Would providing the formData prop improve the readability of this test?